lib/modules/manager/buildkite/extract.ts

```diff
@@ -13,7 +13,7 @@ export function extractPackageFile(content: string): PackageFile | null {
     for (const line of lines) {
       // Search each line for plugin names
       const depLineMatch = regEx(
-        /^[\s-]*(?<depName>[^#\s]+)#(?<currentValue>[^:]+)/
+        /^[\s-?]*(?<depName>[^#\s]+)#(?<currentValue>[^:]+)/
```

Comment:
The `-` character has special meaning when it is between two other characters inside `[]`, so prefer to add it at the beginning or end of the character group